### DISCUSSION TITLE
ChatGLM3_6B工具调用模式支持model.generate()形式调用吗？

### DISCUSSION THREAD

**Discussion by FanZhang91** [zai-org/ChatGLM3#1200] [Q&A] [answered]
使用工具调用时，官方给出的调用方式为：model.chat(tokenizer, query, history=history)。但是chat函数内部会增加额外的prompt信息相关的操作，有没有等价的model.gernerate()方法来实现工具调用？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
不支持
